src/azure-cli/azure/cli/command_modules/storage/_help.py

```diff
@@ -1671,6 +1684,14 @@
       text: az storage fs access set --group 68390a19-a897-236b-b453-488abf67b4dc -p dir -f myfilesystem --account-name mystorageaccount --account-key 0000-0000
 """
 
+helps['storage fs access set-recursive'] = """
+type: command
+short-summary: Set the Access Control on a path and sub-paths in Azure Data Lake Storage Gen2 account.
+examples:
+    - name: Set the Access Control on a path and sub-paths in Azure Data Lake Storage Gen2 account.
+      text: az storage fs access set-recursive --acl "default:user:21cd756e-e290-4a26-9547-93e8cc1a8923:rwx" -p dir -f myfilesystem --account-name myadlsaccount --account-key 0000-0000
```

Comment:
The format of the `--acl` parameter is `[scope:][type]:[id]:[permissions]`, but the location of each part in the `acl` parameter seems to be a bit confusing. Could we describe it in more detail?
For example:
When `permissions` are not passed in for `acl` parameter in the first example, the input value is `--acl "default:user:21cd756e-e290-4a26-9547-93e8cc1a8923"`, but when there are only `type` and `permissions` for `acl` parameter, the input value is `--acl "user::r-x"`

Reply:
Actually it is from SDK and it is hard to see more here. But we could provide a link about acl here.